service/templates/base_angular.html

```diff
@@ -57,7 +57,7 @@
         </style>
         <script type="text/javascript">
         {{if .ProjectData}}
-            {{if .ProjectData.Project}}
+            {{if .ProjectData.ProjectRef}}
```

Comment:
does the project ref actually have this?

Reply:
Identifier and DisplayName? Yes. We were always using the ProjectRef, but we were checking if Project exists, which is a bug.